client/src/actions/authActions.js

```diff
@@ -12,3 +15,27 @@ export const submitEmail = (userData, history) => (dispatch) => {
       }),
     );
 };
+
+export const loginWithEmail = (payload) => {
+  return (dispatch) => {
+    console.log('payload', payload);
+    return axios.post('/api/users/signup', payload, {}).then(response => {
```

Comment:
looks great now, only one small change
currently we have only `/api/auth/authenticate` for both login & signup with email and password
I would like to review this and maybe split them up, we can also provide more data when signing up first time perhaps.